account_reset_chart/models/res_company.py

```diff
@@ -163,4 +152,30 @@ def unlink_from_company(model):
         unlink_from_company('account.tax.code')
         unlink_from_company('account.journal')
         unlink_from_company('account.account')
+
+        try:
+            self.env['account.partial.reconcile'].search(
+                [('company_id', '=', self.id)]
+            ).unlink()
+        except KeyError:
+            pass
+
+        try:
+            self.env['account.full.reconcile'].search([]).unlink()
```

Comment:
Hrmmm I could have sworn that there were stragglers when I originally made this, but I just retested and it seems fine without. Good call